dll/win32/shell32/stubs.cpp

```diff
@@ -451,26 +451,38 @@ SheSetCurDrive(INT iIndex)
     return 1;
 }
 
-/*
- * Unimplemented
- */
 EXTERN_C LPWSTR
 WINAPI
-SheRemoveQuotesW(LPWSTR lpInput)
+SheRemoveQuotesW(LPWSTR psz)
 {
-    FIXME("SheRemoveQuotesW() stub\n");
-    return NULL;
+    PWCHAR pch;
+
+    if (*psz == L'"')
```

Comment:
```suggestion
    if (psz && *psz == L'"')
```
Wouldn't your initial implemenation crash when being called with a nullptr? I would recommend to add a test with NULL being passed also.

Reply:
The original function doesn't check it.

Reply:
So you say if you pass a nullptr to that func in Windows it will crash?
Does that still apply to Windows 10?

Reply:
Added `NULL` tests.
Confirmed in Win2k3 and WinXP.
Win10 skips the tests.

Reply:
I don't think we should follow Microsofts behavior in crashing to such an extent. I do really vote for checking for NULL here. I don't think this would break any compatibility. I don't think there is any application out there that requires to crash "to behave correctly".

Reply:
It should be "correctly crashing". That is the expected behavior.

Why does every PC crash on system failure? To stop the wrong process. To find the fatal mistake(s) on the system.